changelogs/fragments/9959-update-opennebula-onevm-updateconf-params.yml

```diff
@@ -0,0 +1,11 @@
+minor_changes:
+  - |
+    Update `UPDATECONF_ATTRIBUTES` in `one_vm.py` to include new parameters as per the latest OpenNebula API documentation.
+    Added parameters:
+    - OS: FIRMWARE
+    - CPU_MODEL: MODEL, FEATURES
+    - FEATURES: VIRTIO_BLK_QUEUES, VIRTIO_SCSI_QUEUES, IOTHREADS
+    - GRAPHICS: PORT, COMMAND
+    - VIDEO: ATS, IOMMU, RESOLUTION, TYPE, VRAM
+    - RAW: VALIDATE
+    - BACKUP_CONFIG: FS_FREEZE, KEEP_LAST, BACKUP_VOLATILE, MODE, INCREMENT_MODE
```

Comment:
```suggestion
    one_vm - update allowed values for ``updateconf`` to include new parameters as per the latest OpenNebula API documentation.
    Added parameters:

    * ``OS``: ``FIRMWARE``;
    * ``CPU_MODEL``: ``MODEL``, ``FEATURES``;
    * ``FEATURES``: ``VIRTIO_BLK_QUEUES``, ``VIRTIO_SCSI_QUEUES``, ``IOTHREADS``;
    * ``GRAPHICS``: ``PORT``, ``COMMAND``;
    * ``VIDEO``: ``ATS``, ``IOMMU``, ``RESOLUTION``, ``TYPE``, ``VRAM``;
    * ``RAW``: ``VALIDATE``;
    * ``BACKUP_CONFIG``: ``FS_FREEZE``, ``KEEP_LAST``, ``BACKUP_VOLATILE``, ``MODE``, ``INCREMENT_MODE``.

    (https://github.com/ansible-collections/community.general/pull/9959).
```

Reply:
@felixfontein will the changelog tool produce multilined content (like the bullet list) above? I thought the output there would always be a single line.

Reply:
More importantly: do we want multilined content in the changelogs? My gut feeling tells me we should avoid that, but I got no strong argument either side.

Reply:
> @felixfontein will the changelog tool produce multilined content (like the bullet list) above? I thought the output there would always be a single line.

Yes, that works fine. You can even have multiple paragraphs.

> More importantly: do we want multilined content in the changelogs? My gut feeling tells me we should avoid that, but I got no strong argument either side.

That's a very good question. I'm personally also not very happy about it.

Maybe this information should be listed in the module's documentation (including existing values), instead of the changelog? That's generally more helpful for users anyway.

Reply:
It should *definitely* be in the docs, and I also think that there should a comment about them being added in 10.6.0